blackjack.rb

```diff
@@ -93,24 +90,23 @@ def player_victory
   end
 
   def dealer_score
-    dealer_hand.reduce(0) { |sum, card| sum += card.value }
+    dealer_hand.reduce(0) { |a, e| a += e.value }
   end
 
   def check_blackjack
     if player_score == 21
-      puts "Player wins with Blackjack"
-      player_victory
+      puts 'Player wins with Blackjack'
+      player_wins?
+      resolution
     elsif dealer_score == 21
-      puts "Dealer wins with Blackjack"
-      `say "Better luck next time"`
-      dealer_victory
+      puts 'Dealer wins with Blackjack'
+      `say 'Better luck next time'`
+      player_wins?
       resolution
     end
   end
 
-
-  def player_wins?
-
+  def player_wins? # too long, too complex
```

Comment:
Hey James!

The comments were placeholders (and notes to me) based on Rubocop's code
analysis. I should have removed them before sending you the pull request.

As to the hitting, I didn't know what you meant at first - I made super
certain that the player could hit before putting in the PR. But I see that
you mean the dealer can't hit, and you're right. Sorry about that. No ideas
off the top of my head.

We can take a look at it today if you like

On Mon, Mar 14, 2016 at 7:38 PM, JHenry <notifications@github.com> wrote:

> In blackjack.rb
> <https://github.com/jajajames/command_line_blackjack/pull/5#discussion_r56094988>
> :
>
> >        resolution
> >      end
> >    end
> >
> > -
> > -  def player_wins?
> > -
> > +  def player_wins? # too long, too complex
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jajajames/command_line_blackjack/pull/5/files#r56094988>
> .
>



-- 
Ryder Timberlake
Developer, Musician, Teacher
wrtimberlake@gmail.com